modules/osdk-updating-131-to-1361.adoc

```diff
@@ -58,7 +58,7 @@ ifdef::helm[]
 .Example Dockerfile
 [source,docker,subs="attributes+"]
 ----
-FROM registry.redhat.io/openshift4/ose-helm-rhel9-operator:v{osdk_ver} <1>
+FROM registry.redhat.io/openshift4/ose-helm-rhel9-operator:v{product-version} <1>
 ----
 <1> Update the Helm Operator version from `{osdk_ver_n1}` to `{osdk_ver}`.
```

Comment:
We should update here also. Probably to this?
```suggestion
<1> Update the Helm Operator version to `{product-version}`.
```

Reply:
As well as line number 56 and 66.

Reply:
You're right, these have gotten kind of Frankenstein'd by this point, so I've cleaned up both the Helm and Ansible steps now to better match each other for style, including removing the callout from the Helm one.